### DISCUSSION TITLE
PCL2 血泪史：2020~2023 TZG 事件，至今还没有结束

### DISCUSSION THREAD

**Discussion by PlayMcBKuwu** [Meloong-Git/PCL#2132] [讨论]
# 开端
这个本在石墨有文档（比我搬运的完整一些）：https://shimo.im/docs/xWR3wT3cqxcYt8PQ/read
本来在知乎也有专栏的，甚至龙猫在知乎曾**说过不会删除这篇专栏**。~~（虽然最后因为 TZG 贩剑以“侵犯个人名誉权” 举报，被知乎官方删除，龙猫申诉要求恢复无果还是没留下来）~~
以至于今天，大多数人知道的 PCL2 血泪史只剩下 **Hiper 事件** 了。
后来，在哔哩哔哩当了下搬运工，写了篇专栏：https://www.bilibili.com/read/cv24602954
具体什么事情，自己看吧 www
***
# 现在
令我感到奇怪的是，**龙猫自 2021 年 8 月以后再也没有提起或在石墨文档更新与 TZG 有关的事件**。
**许多网友搬运了 TZG 事件，但也在 2021 年左右纷纷删除。**
现在，能在网上找到的，不多了。~~（互联网档案馆除外，但大陆进不去）~~
## 但这件事，远远没有结束。
***
如今，在 Github 上，**仍有 PCL2 的破解 / 解锁项目。**~~（甚至今年 5/19 还更新了一次）~~
如今，TZG 的账户，在 bilibili，仍然没有封号。（无法确定是否换号主）
如今，“PCL2 免费交流群”中，**疑似仍然有 PCL2 内测版的外传。**
### 但我的目的，即为让更多人知道那遥不可及的事实。
***
**P.S.:本来发现了 TZG 的新博客地址但是历史记录找不到了 www 没记错的话 5 月份还在更新**
~~不知道谁能帮忙找一下~~
***
**P.S.:曾经有个梗叫做 Jvav，也是 TZG 弄出来的，可惜的是 TZG 侵害未成年人肖像权（虽说后来他在知乎补了“已经过允许”，但不见得是真的，名字都是开别人盒来的这种人能信么），成功的喜提知乎永久封禁大礼包**
***
隔壁讨论 TZG 的论坛内容 **（需要非中国大陆网络连接）** ：https://hostloc.com/thread-687435-1-1.html
貌似讨论的 **有点过头**（雾）
无恶意，只是搬运一下 :(

**Comment by LTCatt** [comment]
知乎的专栏和很多相关回答/文章都被 TZG 以 “侵犯个人名誉权” 举报，被知乎官方删除，申诉要求恢复无果 :/
我自己倒是不会删……

**Reply by PlayMcBKuwu** [reply]
这点我挺感谢互联网档案馆的 ~~（还有一个原因：TZG 的隐藏主题解锁虽然存档了但是打不开）~~

**Reply by iPodtouch6557** [reply]
说实话他还侵犯了你的权益呢（

**Reply by LTCatt** [reply]
补充进去了：https://shimo.im/docs/xWR3wT3cqxcYt8PQ#anchor-IMrA

**Reply by LTCatt** [reply]
以及……是的，这件事是 PCL 决定不添加任何第一方网络服务的原因，也是 PCL 快照版采用特殊更新机制和加壳（导致快照版被报毒）的原因……

**Comment by so1ve** [comment]
建议挂起来以免后来的人不知道为什么pcl没有服务器（摊手）

**Reply by PlayMcBKuwu** [reply]
这种事情只能说随缘吧，能看到的即看到，看不到的...qwq
这就是为啥 2020 年那段时间 PCL2 更新还不需要去蓝奏云

**Reply by so1ve** [reply]
也是

**Reply by iPodtouch6557** [reply]
所以为啥没服务器

**Reply by PlayMcBKuwu** [reply]
自己看 https://www.bilibili.com/read/cv24602954
完整版 https://shimo.im/docs/xWR3wT3cqxcYt8PQ/read

**Reply by iPodtouch6557** [reply]
哦，之前那个“不会制作服务器相关功能”就是打这来的吧

**Reply by PlayMcBKuwu** [reply]
对，龙猫被人坑惨了

**Reply by iPodtouch6557** [reply]
学生党真惨啊，为了生活跟别人掰扯来掰扯去……祝龙猫好运;.(

**Reply by iPodtouch6557** [reply]
我2023年才赞助的，请允许我可能有点冒犯的好奇，请问那段时间他的个签是啥
~我的个签一直是“qq签名是没有用滴，应该丢进垃圾桶里面”~

**Reply by PlayMcBKuwu** [reply]
最好别好奇 ~~因为我也不知道，我只知道肯定是很致郁的~~

**Reply by iPodtouch6557** [reply]
其实我也不大敢好奇，所以问的很谨慎
github啥时候来了新特性会把在楼中楼quote的回复叠加到发普通回复的文本框里

**Reply by so1ve** [reply]
bug :( 去 https://github.com/community/community 反馈

**Reply by iPodtouch6557** [reply]
目前不打算去反馈，近几天身边一直没有电脑，可能是我手机Safari浏览器的问题（
如果我回了家发现电脑也有问题再考虑反馈

**Reply by so1ve** [reply]
我手机版最新版chrome也有这个问题，好像已经有人反馈了

**Reply by PlayMcBKuwu** [reply]
巧了 Windows 8.1 和 Windows 10 的 Microsoft Edge 114.0.1823.58 (正式版本) (64 位) 都会这样

**Reply by iPodtouch6557** [reply]
所以我那个问题是不是跑偏了，咋都变成bug的事了（（（

**Comment by Chiloven945** [comment]
总感觉对这件事情有点印象（

**Reply by WForst-Breeze** [reply]
这事稍微有点久远了，前面的几批 PCL2 用户应该都差不多了解一点，第一个 Beta 版没记错的话就是受这个事件影响才问世的（Beta 1.3.4）
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/72a2577d-5796-4dbe-a2e0-450f4297e643)

**Reply by iPodtouch6557** [reply]
唉，这事儿的影响有点久远啊…………

**Reply by wuliaodexiaoluo** [reply]
beta在哪下。。。。

**Reply by allMagicNB** [reply]
> beta在哪下。。。。

爱发电动态

**Reply by wuliaodexiaoluo** [reply]
我不知道怎么说，但是翻过去都是发电2.33、6.66、23.33 解锁
问题是我高达77.84的赞助解锁不了

**Reply by allMagicNB** [reply]
> 我不知道怎么说，但是翻过去都是发电2.33、6.66、23.33 解锁
> 问题是我高达77.84的赞助解锁不了

温馨提示：发电之后只有一个月时间可以查看之前的动态，这玩意是会过期的¯\\_(ツ)\_/¯
而且公开预览版我记得好像不需要发电解锁吧？

**Reply by wuliaodexiaoluo** [reply]
找不到 Beta ，因为动态实在是太多了

**Comment by PlayMcBKuwu** [comment]
# 待った！「等一下！」
## 就这么结束了吗？
> 不得不感慨 [Google](https://www.google.com/search?q=thezihangu&newwindow=1&sca_esv=575429130&cs=0&sxsrf=AM9HkKlCqEdtwm3DYeQxfWJOqGQvLCDhOg:1697885167392&filter=0&biw=1229&bih=626&dpr=2#ip=1)（需要非中国大陆网络，这个链接是 Google 搜索 TheZihanGu 的信息，~~能有 2,250 个搜索结果我也真是服了~~） 的能力了，几乎所有 TZG 干过的坏事 ~~和他剩余未知的信息~~ 都有了
> Google 能查到的实在太多，各位挂上梯子慢慢看，这里只总结部分。
> ~~如果我实在做的太过火了还请提醒我，我怕屠龙者终成龙，从揭秘真相的一个热心网友，变成臭名远扬的开盒者。~~
# TZG 的知乎“未死”？？？
- 好吧，之前我说的实在太过头了，TZG 在知乎根本没被封禁，也就一个认证被删了
- https://www.zhihu.com/people/zihangu
- 甚至还能看到 [TZG 说](https://www.zhihu.com/column/c_1255922645968838656)，里面的”[对于 PCL 2 鉴定盗版和检测非法隐藏主题解锁的查看结果](https://zhuanlan.zhihu.com/p/347828430)“实在是一时不知道该怎么说，各位自己看吧。
# TZG 的 Steam 账户、Github 和 CSDN（貌似用处不大）
- Github 一直在咕咕咕，啥也没发，不道是不是全删了：https://github.com/TheZihanGu
- ~~他 Steam 账户注销了~~
- CSDN：https://blog.csdn.net/u010380905
# 重头戏之 TZG 的 Telegram
- https://tgstat.com/channel/@thezihangu
- 需要非中国大陆网络
- ~~疑似人格分裂症，根本不知道是不是本人。~~
- ~~还有一个可怜的姓顾的人的身份证号和本名~~
- ~~在 Google 找的话，你会发现他的 Telegram 经常光顾 **机场**，**簧篇** 等内容，还举报龙猫群讨论梯子，真是五十步笑百步啊。~~
### TZG 曾开过的 Minecraft 服务器
- https://www.bilibili.com/video/av50044107/
# TZG 在互联网上的信息
- ~~别找了，没有那么过火的总结。~~
- 你会发现，他几乎在每个地方都留下了足迹并注册了帐户，pixiv，萌娘百科，比比皆是 ~~，时不时还打打歌~~。
- ~~不过几乎在 2020 年后都把信息删的差不多了，几乎只剩账户本身，没错，我只想说这个。~~
- [曾经甚至写过开服教程](https://guide.minecraft.kim/)
- [龙猫曾经还给过 TZG 《古墓丽影》可还行（](https://afdian.net/p/43f7d4a298d311e992fe52540025c377)

### ~~不计他干的坏事的话，其实他人真的还挺好的。~~
### ~~主要是他在各大论坛倒的确乐于助人就是了。~~

**Reply by iPodtouch6557** [reply]
电脑没VPN手机有 不过近期没时间，回头看
但是不建议你继续补充信息了，再往后就不可避免会变成kh的（）

**Comment by Silverteal** [comment]
这里没有提到一件事：当时的PCL会扫描腾讯QQ的账号文件夹，并把检测到的所有QQ号上传到服务器，以探测异常使用情况。在当时隐私政策还没有完全产生的情况下。这显然违反了个保法或者其他的什么。这也是当事人对PCL发起声讨的一点（而且我认为确实应该声讨）。后来随着解锁码的应用，这个功能就下线了。

**Reply by iPodtouch6557** [reply]
确实有道理，同时这是不是全部原因？

**Comment by Cheng2024666** [comment]
pcl2的交流群是什么啊

**Reply by tangge233** [reply]
PCL 还没有面向全部用户的交流群

**Reply by Joker2184** [reply]
> pcl2的交流群是什么啊

内群.jpg

**Comment by Open-KFC** [comment]
我随便选的头像怎么和这个人一样啊
幼犬酱的粉丝里有坏人

**Reply by Open-KFC** [reply]
想来想去还是这张最好看……

**Reply by LinQingYuu** [reply]
> 我随便选的头像怎么和这个人一样啊 
>幼犬酱的粉丝里有坏人

一开始看到这头像给我看蒙了，然后翻了翻历史浏览记录发现这不是 TZG 的头像么（摊手

**Reply by wyc-26** [reply]
wc 我就说怎么有那么一丝丝熟悉
